insights/parsers/systemctl_status.py

```diff
@@ -0,0 +1,86 @@
+"""
+
+/bin/systemctl status <Services_Name> - Command
+===============================================
+
+This command will show status of all the, the status could be active, inactive,
+enabled or disabled. If the service state is disabled then service will not
+start after reboot, and state will be inactive disabled.
+
+"""
+# coding: utf-8
+import os
+
+from .. import LegacyItemAccess
+from .. import Parser
+from .. import parser
+
+
+class SystemctlStatus(LegacyItemAccess, Parser):
+    """ Class for parsing ``/bin/systemctl status <Service_Name>`` """
+
+    def parse_content(self, content):
+        """
+        Sample Input::
+
+            openvswitch-nonetwork.service - Open vSwitch Internal Unit
+                Loaded: loaded (/usr/lib/systemd/system/openvswitch-nonetwork.service; static; vendor preset: disabled)
+                Active: inactive (dead) since Fri 2017-04-14 00:23:48 UTC; 3min 17s ago
+                Main PID: 673 (code=exited, status=0/SUCCESS)
+
+        Sample Output::
+
+            {'Loaded' : 'openvswitch-nonetwork.service',
+            'vendor preset' : 'disabled',
+            'Active' : 'inactive',
+            'Main PID' : '673'}
+        """
+
+        self.data = {}
+        for line in content:
+            if "Main PID" in line or "Loaded" in line or "Active" in line:
+                line = line.replace('   ', '')
+                for spch in [';', '(', ')']:
+                    line = line.replace(spch, '\n')
+                for st in line.split('\n'):
```

Comment:
You might consider using `re.split` replacing the above four lines with something like:
```
for st in re.split('[;\(\)\s]*', line):
```
The above example may not produce exactly what you need.